lib/geoservices/queryable.rb

```diff
@@ -14,7 +14,16 @@ def query(layer_idx=0, options={})
     # returns nil if the layer does not have the Capability: Query
     def count(layer_idx=0, options={})
       return nil unless self.metadata["capabilities"] =~ /Query/
-      query(layer_idx, options.merge(:returnCountOnly => true))
+      # Current Version is not returned at lower than 10.1
+      unless self.metadata['currentVersion'].nil?
```

Comment:
current version is going to be nil, not less than 10.1 on versions we need this check for
![image](https://cloud.githubusercontent.com/assets/7832202/7420716/c235a130-ef4c-11e4-98fa-b9636636ff2a.png)